src/Compiler/pars.fsy

```diff
@@ -2375,8 +2375,23 @@ objectImplementationMember:
      { let rangeStart = rhs parseState 1
        $3 $1 $2 rangeStart }
 
-  | opt_attributes staticMemberOrMemberOrOverride error
-      { [] }
+  | opt_attributes staticMemberOrMemberOrOverride recover
+      { let rangeStart = rhs parseState 1
+        let memFlagsBuilder, leadingKeyword = $2
+        let flags = Some(memFlagsBuilder SynMemberKind.Member)
+        let xmlDoc = grabXmlDocAtRangeStart (parseState, $1, rangeStart)
+        let trivia = { LeadingKeyword = leadingKeyword; InlineKeyword = None; EqualsRange = None }
+        let mMember = rhs parseState 3
```

Comment:
Please don't use the range from `recover` rule here, as it may include the next token that is outside the member node. It would lead to two disjoint nodes both containing this token (which we don't want). In similar rules I've used an empty range at the last matched rule, something similar can be done here.

Reply:
will fix this in next week

Reply:
fixed